WADSuite/Button.cs

```diff
@@ -0,0 +1,127 @@
+﻿//******************************************************************************
+//
+// Copyright (c) 2016 Microsoft Corporation. All rights reserved.
+//
+// This code is licensed under the MIT License (MIT).
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+// THE SOFTWARE.
+//
+//******************************************************************************
+
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using OpenQA.Selenium.Appium.Windows;
+using System.Threading;
+
+namespace UWPControls
+{
+    [TestClass]
+    public class Button : Test_Base
+    {
+        private static WindowsElement buttonElement = null;
+
+        [ClassInitialize]
+        public static void ClassInitialize(TestContext context)
+        {
+            Setup(context);
+            //NavigateTo("Buttons", "Button");
+            var buttonTab = session.FindElementByName("Basic Input");
+            buttonTab.Click();
+            var button = session.FindElementByName("Button");
+            button.Click();
+            Thread.Sleep(1000);
```

Comment:
Sleepy threads are a code smell. Why do we need this, and can we avoid it?

Reply:
WinAppDriver does not support internal wait timers (e.g. explicitWait) at the moment - this is a backlog item to implement in the future. Until then, Thread.Sleep timers are the simplest solution - but agreed, we would want to replace this eventually.